tools/ShapeChecker/build.xml

```diff
@@ -122,4 +126,17 @@
             <arg value="-V"/>
         </java>
     </target>
+
+    <target depends="makejar" name="OSLC AM Check" description="Run ShapeChecker on OSLC Asset Management">
```

Comment:
I've added a comment to my Ant build.xml file to explain why that Asset Management test should not necessarily be added to Circle CI, and in this case, we really don't want that, as it would fail - as far as I know, no one has worked on Asset Management and it has several issues that would need to be addressed.